cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/Action.hs

```diff
@@ -41,14 +40,11 @@ hprop_golden_governanceActionCreateConstitution =
 
     goldenActionFile <-  H.note "test/cardano-cli-golden/files/golden/governance/action/create-constitution-for-stake-address.action.golden"
 
+    -- Remove cbor hex from comparison, as it's not stable
     H.redactJsonField "cborHex" "<cborHex>" actionFile redactedActionFile
 
     H.diffFileVsGoldenFile redactedActionFile goldenActionFile
 
-    H.assertFileOccurences 1 "Governance proposal" actionFile
-
-    H.assertEndsWithSingleNewline actionFile
-
```

Comment:
This can be removed, because we actually do a nearly-full (only the cbor Hex gets ignored) comparison against a golden file.